modules/engage-paella-player/src/main/resources/OSGI-INF/paella-config-file.xml

```diff
@@ -11,5 +11,5 @@
     <provide interface="org.opencastproject.engage.paella.PaellaConfigRest" />
   </service>
   <reference name="security-service" interface="org.opencastproject.security.api.SecurityService"
-    cardinality="1..1" policy="static" bind="setSecurityService" />
```

Comment:
Just wondering: Do we start to use the OSGI defaults (1..1 & static)? Just because those defaults seem to be explicitly set in many, many places...

Reply:
We have them set explicitly in many places that's true. But we also haven't set them explicitly in many cases ;-P

I've adjusted this since I was working on this anyway.

Reply:
Fine for me.